templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -3,6 +3,23 @@ kind: Template
 metadata:
   name: ansible-service-broker
 objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: dr
```

Comment:
For someone looking at this Service and wondering what it is, could this be a touch more descriptive? It'll save us from answering the question "What is the 'dr' service?" a hundred times. ;)